### PR TITLE
files: add statistics for pricing

### PR DESCRIPTION
**Pull request by jma**
* Injects the file library pid in to the document.
* Adds number_of_docs_with_files, number_of_files, files_volume princing stats.

### CODE REVIEW

File: tests/api/documents/test_documents_rest.py [hunk 1]
```diff
@@ -682,7 +683,7 @@ def failure(response_data):
 
     # STEP#1 :: SEARCH FOR AN EXISTING IDENTIFIER
     #   Search for an existing encoded document identifier. The ISBN-13 is
-    #   encoded into document data. Search on this specific value will return
+#   encoded into document data. Search on this specific value will return
```
Comment (PascalRepond) [inline tests/api/documents/test_documents_rest.py:686]: Tab deleted by mistake?

**Review by PascalRepond — APPROVED** [review]

File: rero_ils/modules/documents/dumpers/indexer.py [hunk 2]
```diff
@@ -240,6 +240,8 @@ def _process_files(self, record, data):
         files = {}
         for record_file in record.get_records_files():
             collections = record_file.get('metadata', {}).get('collections')
+            owners = record_file.get('metadata', {}).get('owners')
+            library_pid = owners[0].replace('lib_', '')
```
Comment (lauren-d) [inline rero_ils/modules/documents/dumpers/indexer.py:244]: this raise an exception if `owners` is `None`

**Review by lauren-d — APPROVED** [review]

**Review by rerowep — APPROVED** [review]

File: rero_ils/modules/documents/mappings/v7/documents/document-v0.0.1.json [hunk 3]
```diff
@@ -57,6 +57,14 @@
           "rec_id": {
             "type": "keyword"
           },
+          "organisation": {
```
[the remark this answers is not in the document]
Comment (jma) [reply rero_ils/modules/documents/mappings/v7/documents/document-v0.0.1.json:60]: To be consistent with holdings.

File: rero_ils/modules/documents/dumpers/indexer.py (hunk 2 above)
Comment (jma) [reply rero_ils/modules/documents/dumpers/indexer.py:244]: You are right. However for the moment owners should be there. I probably create a new PR at the end of the US and refactor the `links` and `owners` properties.

**Pull request merged**